tests/base/fixtures/checkout.page.ts

```diff
@@ -51,4 +56,53 @@ export class CheckoutPage {
     await expect(this.continueShoppingButton, `Your order number is: ${orderNumber}`).toBeVisible();
     return orderNumber;
   }
+
+  async applyDiscountCodeCheckout(code: string){
```

Comment:
@shayfaber Is there a reason this needs to be *Checkout? Or can we remove `Checkout` from this function name? Because it is already called exported as CheckoutPage. So then it will be CheckoutPage->applyCouponCode.